packages/data/src/plugins/actions.js

```diff
@@ -72,15 +73,15 @@ export function* installPlugins( plugins ) {
 		}
 
 		if ( Object.keys( results.errors.errors ).length ) {
-			throw results.errors;
+			throw results.errors.errors;
 		}
 
 		yield setIsRequesting( 'installPlugins', false );
 
 		return results;
 	} catch ( error ) {
 		yield setError( 'installPlugins', error );
-		throw new Error( formatErrors( error ) );
+		throw new Error( formatErrorMessage( error ) );
```

Comment:
`formatErrors` returns an Object, and I'm thinking this might have just been a misunderstanding of the `Error` API in JS. I've adjusted this to try provide a useful error message instead. Before this change the Error bubbled up to the error handler was put into a snackbar and it just said `[object Object]` so it wasn't very helpful.